Bounding_volumes/include/CGAL/min_quadrilateral_2.h

```diff
@@ -20,7 +20,6 @@
 #include <CGAL/basic.h>
 #include <CGAL/Optimisation/assertions.h>
 #include <iterator>
-#include <boost/bind.hpp>
 #include <boost/function.hpp>
```

Comment:
Replace `<boost/function.hpp>` with `<functional>`, for `std::function`. But, actually, I wonder why you use `std::function` at all. I can be slightly slower than using the lamba directly.

Reply:
We need the type of the lambda for a data member of a class (and in other cases, IIRC, we need `result_type` that is provided by `std::function`.